storybook/pages/DerivationPathInputPage.qml

```diff
@@ -125,6 +125,17 @@ SplitView {
                 font.italic: true
                 color: "red"
             }
+            Label {
+                text: devTxtEdit.warningMessage
+                visible: devTxtEdit.warningMessage.length > 0
+
+                Layout.alignment: Qt.AlignLeft
+                Layout.fillWidth: true
```

Comment:
I guess it should be either Layout.alignment: Qt.AlignLeft or Layout.fillWidth: true